CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.1...HEAD)
 
 ### Backward Compatibility Fixes
+- Use short array syntax `[]` instead of long `array()` - so break compatibility with PHP 5.3
```

Comment:
Which is already in the changelog:

https://github.com/ruflin/Elastica/commit/11bcfe299b1b9a30d51b8dc928bf557f7e6dd7fb